include/mango_v3.hpp

```diff
@@ -42,6 +44,15 @@ const Config DEVNET = {
     {"MNGO", "BTC", "ETH", "SOL", "SRM", "RAY", "USDT", "ADA", "FTT", "AVAX",
      "LUNA", "BNB", "MATIC", "", "", "USDC"}};
 
+const std::unordered_map<std::string, size_t> SERUM_PROGRAM_LAYOUT_VERSIONS = {
+    {"4ckmDgGdxQoPDLUkDT3vHgSAkzA3QRdNq5ywwY4sUSJn", 1},
+    {"BJ3jrUzddfuSrZHXSCxMUUQsjKEyLmuuyZebkcaFp2fg", 1},
+    {"EUqojwWA2rd19FZrzeBncJsm38Jm1hEhE3zsmX3bRc2o", 2},
+    {"9xQeWvG816bUx9EPjHmaT23yvVM2ZWbrrpZb9PusVFin", 3}};
+size_t getLayoutVersion(std::string programId) {
```

Comment:
method not used

Reply:
Forgot to delete, good catch, I was originally planning to have `OpenOrdersV1` and `OpenOrdersV2` according to the serum pk used, as is in the js sdk, but that's unnecessary now.